modules/serverless-kafka-broker-with-isolated-dataplane.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * serverless/eventing/brokers/serverless-using-brokers.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-broker-with-isolated-dataplane_{context}"]
+= Creating a Knative broker for Apache Kafka that uses an isolated data plane
+
+:FeatureName: The Knative Broker implementation for Apache Kafka with isolated data plane
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+The Knative Broker implementation for Apache Kafka has 2 planes:
+
+Control plane:: Consists of controllers that talk to Kubernetes API, watch for custom objects and manage the data plane.
+
+Data plane:: The collection of components that listen for incoming events, talk to Apache Kafka and  send events to the event sinks. The Knative Broker implementation for Apache Kafka data plane is where the events flow, and consists of `kafka-broker-receiver` and `kafka-broker-dispatcher` deployments.
+
+When using a Broker class of `Kafka`, the Knative Broker implementation for Apache Kafka uses a shared data plane. This means that the `kafka-broker-receiver` and `kafka-broker-dispatcher` deployments in the `knative-eventing` namespace are used for all Apache Kafka Brokers in the cluster.
```

Comment:
```suggestion
When it uses a Broker class of `Kafka`, the Knative Broker implementation for Apache Kafka uses a shared data plane. This means that the `kafka-broker-receiver` and `kafka-broker-dispatcher` deployments in the `knative-eventing` namespace are used for all Apache Kafka Brokers in the cluster.
```
- Not sure what or who is `using a Broker class` here. If it is the user, then the sentence has a misplaced modifier and needs to be revised to indicate who or what is doing the "using". If it is the "implementation" that is `using a Broker class`, then consider rewording as I suggest to make this clear.